renderer/sources.gni

```diff
@@ -46,7 +46,7 @@ if (enable_speedreader) {
 
 if (enable_playlist) {
   brave_chrome_renderer_deps += [
-    "//brave/components/playlist:playlist_features",
+    "//brave/components/playlist/common:features",
     "//brave/components/playlist/renderer",
```

Comment:
`//brave/components/playlist/renderer` and `//brave/components/playlist/browser` should depend publicly on `//brave/components/playlist/common`, then you only need to depend on `browser` or `renderer` targets (without explicit `common`).